src/surfaceComponent/Card/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import { PropsWithChildren } from "react";
+import * as React from "react";
+import imageUrl from "../../static/images/card/card.png";
+import * as styles from "./Card.module.css";
+
+interface CardProps {
+  fullWidth?: boolean;
+}
+
+export const CardImage = () => {
+  return (
+    <div className={styles.imageWrapper}>
+      <img src={imageUrl} alt="placeholder" />
+    </div>
+  );
+};
+
+export const CardContent = (props: PropsWithChildren<never, never>) => {
+  return (
+    <div className={styles.cardContent}>
+      <div>
```

Comment:
**nitpick:** You hardcoded it again. The structure of this card should look like following:

```
<CardContent>
 <h3>Card Name</h3>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet
          architecto, autem distinctio dolorem, dolores, esse facere incidunt
          odio officia quasi similique sunt totam unde ut veritatis. A, amet
          aperiam architecto assumenda aut autem commodi consequatur deleniti
          dolorem enim esse excepturi harum illum iure labore minus, modi
          molestiae natus nesciunt non recusandae rem saepe sapiente sint
          temporibus ut vero voluptate. Consequuntur ducimus eius et praesentium
          recusandae. Dolore eaque eius eum excepturi expedita in ipsum laborum
          maiores necessitatibus perspiciatis? Autem beatae, culpa delectus
          dignissimos, dolorem doloremque eaque hic nemo nobis omnis possimus
          quae sint, sit vel veniam! Dignissimos doloribus laudantium provident
          veritatis.
        </p>
</CardContent>
```

`CardContent` component should only set proper CSS for inserted children. You don't need to put anything inside like this.